packages/main/src/AvatarGroup.ts

```diff
@@ -152,7 +149,6 @@ type AvatarGroupClickEventDetail = {
 * click/tap or by using the Enter or Space key.
 * @param {HTMLElement} targetRef The DOM ref of the clicked item.
 * @param {boolean} overflowButtonClicked indicates if the overflow button is clicked
```

Comment:
- remove type tag
- remove type and readonly tags from getter. add explicit return type
- _getWidthToItem remove type from param tag